### PR TITLE
Support prettified bitcoin signing message

### PR DESCRIPTION
**Pull request by Kailai-Wang**
### Context

As topic.

This PR adds a preliminary implementation of prettified bitcoin message similar to prettified ethereum message:  `Litentry authorization message: ` + `0x` prefixed hex-encoded message.

### CODE REVIEW

File: tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts [hunk 1]
```diff
@@ -91,6 +91,7 @@ export default {
                 Ethereum: "EthereumSignature",
                 EthereumPrettified: "EthereumSignature",
                 Bitcoin: "BitcoinSignature",
+                BitcoinPrettified: "BitcoinSignature",
```
Comment (jonalvarezz) [inline tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts:94]: Any strong reason to keep both?

**Review by jonalvarezz — APPROVED** [review]
sweet!

File: tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts (hunk 1 above)
Comment (Kailai-Wang) [reply tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts:94]: Analog to ethereum sig,  we keep both the standard and prettified version

Comment (jonalvarezz) [reply tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts:94]: ethereum standard isn't used by IDH, so I thought of keeping the prettified version only. Enforcing prettified version for clients sounds good to me. we want all users to be able to read what clients ask them to sign.

cryptic signatures (standard versions) could be abused by clients we don't control.

**Review by kziemianek — APPROVED** [review]

File: tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts (hunk 1 above)
Comment (Kailai-Wang) [reply tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts:94]: >  cryptic signatures (standard versions) could be abused by clients we don't control.

I don't get this one

IMO enforcing prettified version doesn't help much here, the client could still wrap it with prefix and trick users into signing it. Incidentally, the prettified version is only related to the identity verification, in trusted service invocation later it accepts standard crypto sig only.

**Pull request merged**